bitnami/opensearch/templates/ingest/hpa.yaml

```diff
@@ -0,0 +1,52 @@
+{{- /*
```

Comment:
If we are adding HPAs, it may make sense to add VPAs like we are doing in new charts

Reply:
We should add it to the chart template also. WIP

Reply:
Added [12d12e4](https://github.com/bitnami/charts/pull/17509/commits/12d12e48101758a316f65abdb21df096b90612d2)